.env.dev

```
@@ -1,6 +1,6 @@
 APP_NAME=CDash
 APP_ENV=testing
-APP_KEY=
+APP_KEY=DV5SQLqXCbpnme4z2pKNujd6gFW9hrj5
 APP_DEBUG=true
 APP_URL=http://localhost:8080
```

```suggestion
APP_URL=
```
Leaving this unset allows the URLs to be resolved as relative links. This way, the deployer doesn't need to know the URL of the site at build time and a single image can be deployed at several different URLs

---

`.env.dev` contains preset values for development and testing purposes.  You should use `.env.example` as a starting place for a production deployment.